client/src/App/App.tsx

```diff
@@ -218,6 +219,42 @@ const App: React.FC = () => {
               {...trashPlansListStatusProp}
               component={ConnectedPlansList}
             />
+            <PrivateComponent
+              redirectPath={APP_CALLBACK_URL}
+              disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+              exact
+              path={`${ACTIVE_PLANS_LIST_VIEW_URL}/:planId`}
+              {...plansListProps}
+              {...activePlansListStatusProp}
+              component={ConnectedPlanAssignment}
+            />
+            <PrivateComponent
+              redirectPath={APP_CALLBACK_URL}
+              disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+              exact
+              path={`${DRAFT_PLANS_LIST_VIEW_URL}/:planId`}
+              {...plansListProps}
+              {...activePlansListStatusProp}
+              component={ConnectedPlanAssignment}
+            />
+            <PrivateComponent
+              redirectPath={APP_CALLBACK_URL}
+              disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+              exact
+              path={`${COMPLETE_PLANS_LIST_VIEW_URL}/:planId`}
+              {...plansListProps}
+              {...activePlansListStatusProp}
+              component={ConnectedPlanAssignment}
+            />
+            <PrivateComponent
+              redirectPath={APP_CALLBACK_URL}
+              disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+              exact
+              path={`${TRASH_PLANS_LIST_VIEW_URL}/:planId`}
+              {...plansListProps}
+              {...activePlansListStatusProp}
+              component={ConnectedPlanAssignment}
+            />
```

Comment:
would it still work, if we had a single route that serves a mission assignment view